BaragonUI/app/components/albs/TargetGroupsTable.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+
+import UITable from '../common/table/UITable';
+import Column from '../common/table/Column';
+
+const targetGroupName = (targetGroup) => {
+  const name = targetGroup.targetGroupName;
+  return <Link to={`/albs/target-groups/${name}`}>{name}</Link>;
+};
+
+const TargetGroupsTable = ({targetGroups}) => {
+  return (
+    <div>
+      <UITable
+        data={targetGroups}
```

Comment:
Not off the first request, unfortunately. Amazon really cut back on the information that they give you per-request. [This](http://docs.aws.amazon.com/elasticloadbalancing/latest/APIReference/API_DescribeTargetGroups.html) suggests that you need to use `DescribeTargetHealth` to get the targets for a target group, and it requires that you repeat the request for each target group.